packages/storybook/src/web-components/button-login/button-login.md

```diff
@@ -0,0 +1,154 @@
+
+| Token                                        | Waarde    |
+| :------------------------------------------- | :-------- |
+| `--lux-button-login-background-color`        | `#282828` |
+| `--lux-button-login-border-color`            | `#696969` |
+| `--lux-button-login-color`                   | `#282828` |
+| `--lux-button-login-active-background-color` | `#a7a7a7` |
+| `--lux-button-login-active-border-color`     | `#282828` |
+| `--lux-button-login-active-color`            | `#282828` |
+| `--lux-button-login-hover-background-color`  | `#e6e6e6` |
+| `--lux-button-login-hover-border-color`      | `#696969` |
+| `--lux-button-login-hover-color`             | `#282828` |
+
+#### Size
+
+Schaal het icoon mee met de hoogte van de knop. Houd de knop minimaal 48px hoog in verband met de toegankelijke target size.
+
+| Token                                | Waarde  |
+| :----------------------------------- | :------ |
+| `--lux-button-login-min-inline-size` | `118px` |
+| `--lux-button-login-icon-size`       | `40px`  |
+
+## State
+
+<ul class="section">
+  <li>
+    <lux-button-login agent="digid"></lux-button-login>
+    <span>Default</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--hover" agent="digid"></lux-button-login>
+    <span>Hover</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--active" agent="digid"></lux-button-login>
+    <span>Active</span>
+  </li>
+  <li>
+    <lux-button-login class="sb-state--focus" agent="digid"></lux-button-login>
+    <span>Focus</span>
+  </li>
+</ul>
+
+## Agent
+
+<ul class="section">
+  <li>
+    <lux-button-login agent="digid"></lux-button-login>
+    <span>Digid</span>
+  </li>
+  <li>
+    <lux-button-login agent="digid-machtigen"></lux-button-login>
+    <span>Digid Machtigen</span>
+  </li>
+  <li>
+    <lux-button-login agent="eherkenning"></lux-button-login>
+    <span>eHerkenning</span>
+  </li>
+  <li>
+    <lux-button-login agent="eidas"></lux-button-login>
+    <span>eIDAS</span>
+  </li>
+</ul>
+
+## Language: en
+
+Gebruik de onderstaande Engelse vertalingen door het `label` attribuut toe te wijzen.
+
+```html
+<lux-button-login agent="digid" label="Log in"></lux-button-login>
+```
+
+<ul class="section">
+  <li>
+    <lux-button-login agent="digid" label="Log in"></lux-button-login>
+    <span>Digid</span>
+  </li>
+  <li>
+    <lux-button-login agent="digid-machtigen" label="Log in as representative"></lux-button-login>
+    <span>Digid Machtigen</span>
+  </li>
+  <li>
+    <lux-button-login agent="eherkenning" label="Login as a company or organisation"></lux-button-login>
+    <span>eHerkenning</span>
+  </li>
+  <li>
+    <lux-button-login agent="eidas"></lux-button-login>
+    <span>eIDAS</span>
+  </li>
+</ul>
+
+## Toegankelijkheid
+
+Biedt jouw organisatie meer dan één inlogmiddel aan? Plaats de inlogknoppen in een lijst, zodat een blinde of slechtziende gebruiker van de voorleessoftware te horen krijgt dat er meerdere opties volgen.
```

Comment:
Wij hanteren in onze documentatie de termen 'screenreadergebruikers' en 'toetsenbordgebruikers'. Omdat screenreadergebruikers niet altijd visueel beperkt zijn maar soms ook een taalachterstand hebben oid. Hoewel het gebruik van een lijst in dit geval wel degelijk de blinde of slechtziende gebruikers zal helpen, misschien toch mooi om hier een gelijk in op te trekken?

```suggestion
Biedt jouw organisatie meer dan één inlogmiddel aan? Plaats de buttons dan in een lijst zodat screenreadergebruikers te horen krijgen dat er meerdere opties volgen.
```

Reply:
Goed punt